src/reducers/user.ts

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { SET_USER } from '../actions/user';
+// NEED CONFIRMATION FOR ANY ACTION
+import { AnyAction } from 'redux';
```

Comment:
Reorganise imports.


- External Imports 
- Internal Imports 
- Components
- Styling 

Note: This is not a standard, just a recommended practice to keep large codebases organised.